rdf-star-cg-spec.html

```diff
@@ -394,12 +394,21 @@ <h2>Parsing</h2>
     </section>
     </section>
 
-    <section id="n-triples-star" class="informative">
+    <section id="n-triples-star">
     <h2>N-Quads*</h2>
 
     <p>The [[N-QUADS]] format is extended to describe the N-Quads* format using the same production updates described in the <a href="#n-triples-star-grammar">N-Triples* Grammar</a>.</p>
 
     <p class="note">As RDF* describes <a>embedded triples</a> and not embedded quads, the <a data-cite="N-QUADS#grammar-production-graphLabel">`graphLabel`</a> component of an N-Quads <a data-cite="N-QUADS#grammar-production-statement">`statement`</a> does not apply to the <a href="#grammar-production-nt-embTriple">`embTriple`</a> component.</p>
+
+    <p>An N-Quads* document defines an <a>RDF* dataset</a> composed of
+      a single <a>default graph</a>, which is an <a>RDF* graph</a>,
+      and zero or more <a>named graphs</a>, which are <a>RDF graphs</a>.</p>
+
+    <p>A conforming N-Quads* parser MUST parse any valid
+      <strong>N-Quads document</strong> and additionally parse the <a href="#grammar-production-nt-subject">`subject`</a> and <a href="#grammar-production-nt-object">`object`</a> productions
```

Comment:
```suggestion
      <strong>N-Quads document</strong> and additionally parse the 
      <a href="#grammar-production-nt-subject"><code>subject</code></a> and 
      <a href="#grammar-production-nt-object"><code>object</code></a> productions
```
for easier reading of the source (no need for horizontal scrolling), and to fix the (I think) intended formatting

Reply:
Note that ReSpec understands back-ticks just fine, so it's not necessary to replace with `<code></code>`. But, I'm find with unfolding the long line.